src/AspNet.Security.OAuth.VkId/VkIdAuthenticationHandler.cs

```diff
@@ -0,0 +1,247 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Globalization;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Base64UrlEncoder = Microsoft.AspNetCore.Authentication.Base64UrlTextEncoder;
+
+namespace AspNet.Security.OAuth.VkId;
+
+public sealed class VkIdAuthenticationHandler(
+    IOptionsMonitor<VkIdAuthenticationOptions> options,
+    ILoggerFactory logger,
+    UrlEncoder encoder)
+    : OAuthHandler<VkIdAuthenticationOptions>(options, logger, encoder)
```

Comment:
I can't find the comment where it was originally stated, but please use a non-primary constructor for consistency with the rest of the codebase (adopting them was suggested once, but rejected).